endpoints/openrtb2/auction_test.go

```diff
@@ -1035,6 +1035,41 @@ func TestStoredRequests(t *testing.T) {
 		if !jsonpatch.Equal(newRequest, expectJson) {
 			t.Errorf("Error in processStoredRequests, test %d failed on compare\nFound:\n%s\nExpected:\n%s", i, string(newRequest), string(expectJson))
 		}
+		expectedImp := testStoredImpIds[i]
+		expectedStoredReq := json.RawMessage(testStoredImps[i])
+		if !jsonpatch.Equal(impToStoredReq[expectedImp].StoredImp, expectedStoredReq) {
+			t.Errorf("Error in processStoredRequests, test %d failed on compare stored request\nFound:\n%s\nExpected:\n%s", i, string(impToStoredReq[expectedImp].StoredImp), string(expectedStoredReq))
+		}
+		assert.Equalf(t, testStoreVideoAttr[i], impToStoredReq[expectedImp].EchoVideoAttrs, "EchoVideoAttrs value is incorrect")
+	}
+}
+
+func TestStoredRequestsVideoErrors(t *testing.T) {
+	deps := &endpointDeps{
+		&nobidExchange{},
+		newParamsValidator(t),
+		&mockStoredReqFetcher{},
+		empty_fetcher.EmptyFetcher{},
+		empty_fetcher.EmptyFetcher{},
+		&config.Configuration{MaxRequestSize: maxSize},
+		newTestMetrics(),
+		analyticsConf.NewPBSAnalytics(&config.Analytics{}),
+		map[string]string{},
+		false,
+		[]byte{},
+		openrtb_ext.BuildBidderMap(),
+		nil,
+		nil,
+		hardcodedResponseIPValidator{response: true},
+	}
+
+	for i, requestData := range testStoredRequestsErrors {
+		_, _, errList := deps.processStoredRequests(context.Background(), json.RawMessage(requestData))
+		if len(errList) == 0 {
+			t.Error("processStoredRequests should return error")
```

Comment:
Does t.Error stop the running test - or is that t.Fatal?

Reply:
For some reason I cannot answer to your comment about `resolvedImp`.
At this moment we already have final imp in byte array, but don't have it in object yet.
`resolvedImp, err := jsonpatch.MergePatch(storedImps[impIds[i]], imps[idIndices[i]])` 
We have raw data here that we want to return in response.

Reply:
`t.Error` reports error in logs, but doesn't stop the execution